src/main/osd/osd_elements.c

```diff
@@ -793,13 +793,19 @@ static void osdElementHorizonSidebars(osdElementParms_t *element)
 #ifdef USE_RX_LINK_QUALITY_INFO
 static void osdElementLinkQuality(osdElementParms_t *element)
 {
-    // change range to 0-9 (two sig. fig. adds little extra value, also reduces screen estate)
-    uint8_t osdLinkQuality = rxGetLinkQuality() * 10 / LINK_QUALITY_MAX_VALUE;
-    if (osdLinkQuality >= 10) {
-        osdLinkQuality = 9;
+    // 0-300 or 1-10
+    uint16_t osdLinkQuality = (linkQualitySource == LQ_SOURCE_RX_PROTOCOL_CRSF) ?  (rxGetLinkQuality()  / 3.41) : (rxGetLinkQuality() * 10 / LINK_QUALITY_MAX_VALUE);
```

Comment:
I would refactor this a bit, and integrate the two parts of the ternary into the switch statement below - since the switch statement is already there this will make it a bit easier to read.

Reply:
will  change .. i had thought about it  but then i thought
 it will likely be added back with the other pr for link Quality  for AUX

Reply:
Erm, you haven't done this.

Reply:
I'll look when I get to computer  the switch was remove
More refactoring  ?